tests/i2s_slave_checker.py

```diff
@@ -114,20 +103,21 @@ def run(self):
             xsi.drive_port_pins(self._lrclk, 1)
 
             bclk_frequency = (bclk_frequency_u << 16) + bclk_frequency_l
-            print "CONFIG:bclk:%d in:%d out:%d i2s_justified:%d" % (bclk_frequency, num_ins, num_outs, is_i2s_justified)
-            clock_half_period = float(1000000000)/float(2*bclk_frequency)
+            print(
+                f"CONFIG: bclk:{bclk_frequency} in:{num_ins} out:{num_outs} i2s_justified:{is_i2s_justified}"
+            )
+            clock_half_period = float(1000000000000000) / float(2 * bclk_frequency) # 1fs
```

Comment:
Yes. Adjustments were made to the API in pyxsim compared to xmostest to change the clock reporting from nanoseconds to femtoseconds since the last time this test was rewritten